src/gen-xml.ts

```diff
@@ -962,6 +954,19 @@ function genXmlParagraphProperties (textObj: ISlideObject | TextProps, isDefault
 	return paragraphPropXml
 }
 
+function genXmlBulletMarginIndent (options: ObjectOptions | TextPropsOptions): string {
+	const bullet = typeof options.bullet === 'object' ? options.bullet : {}
+	const bulletMarL = valToPts(bullet.indent ?? DEF_BULLET_MARGIN)
+	const indentLevel = options.indentLevel ?? 0
+
+	if (!bullet.hangingIndentRatio) {
+		return ` marL="${indentLevel > 0 ? bulletMarL + bulletMarL * indentLevel : bulletMarL}" indent="-${bulletMarL}"`
+	}
+
+	const hangingIndent = Math.round(bulletMarL * bullet.hangingIndentRatio)
+	return ` marL="${bulletMarL * (indentLevel || 1)}" indent="-${hangingIndent}"`
```

Comment:
this new behaviour changes the semantic meaning of the text's `bullet.indent` option. Now we use the `indent` directly as the "before text" indent, multiplying by bullet-nesting level, and the amount we choose to offset to the left the "bullet" itself (1./a./i., etc.) is determined by the `hangingIndentRatio`. A value of 1 to the ratio will use PPT's default behaviour when typing bullets, where the first level does not appear indented (see below)

<img width="538" alt="image" src="https://github.com/user-attachments/assets/5bbb88d1-a418-477c-88b1-78c3af7704a6">